backend/services/user.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * UserService that manages the User documents in the DB
+ */
+import UserModel from "../models/user";
+import { InternalError } from "../errors";
+
+export async function registerJob(userId, jobId, owned) {
+    console.debug(`SERVICE: registerJob service running: userId - ${userId}, jobId - ${jobId}, owned - ${owned}`);
+
+    const user = await UserModel.findById(userId);
+    if (!user) {
+        throw InternalError.USER_NOT_FOUND;
+    }
+
+    if (owned) {
+        //TODO Fix use of in operator (use includes instead)
```

Comment:
Looks like you fixed this so this TODO can be removed.

Reply:
Ah yeah! Thanks for pointing it out

Reply:
Will get rid of that TODO line in a future PR